Client/main.go

```diff
@@ -136,7 +137,8 @@ func main() {
 			msg.Text = Bot.ListWeeklyResultByUserId(Id)
 			msg.ParseMode = "HTML"
 		case "chope":
-			msg.Text = "What do you want to order? Tell me the Food ID 😋"
+			msg.Text = "What do you want to order? \nTell me the Food ID 😋 \nEnter -1 to cancel dinner ordering 🙅"
+			//msg.ReplyMarkup = numericKeyboard
```

Comment:
Seems like it is giving warning if we use 🙅‍♀️, ok to change into 🙅?

Reply:
Whats the difference between the two emojis?

Reply:
They have different unicode https://unicode.org/emoji/charts/emoji-list.html
Somehow go thinks the 🙅‍♀️ is beyond unicode range, maybe not yet updated based on unicode list ^